content/shmem_alltoall.tex

```diff
@@ -100,6 +98,25 @@
     If \VAR{team} compares equal to \LibConstRef{SHMEM\_TEAM\_INVALID} or is
     otherwise invalid, the behavior is undefined.
 
+    Before any \ac{PE} calls a \FUNC{shmem\_alltoall} routine, the following
+    conditions must be ensured, otherwise the behavior is undefined:
+    \begin{itemize}
+        \item The \dest{} array on all \acp{PE} in the team is ready to
+            accept the result of the operation.
+        \item The \source{} array at the local \ac{PE} is ready to be
+            read by any \ac{PE} in the team.
+    \end{itemize}
+    The application does not need to synchronize to ensure that the \source{}
+    array is ready across all \acp{PE} prior to calling this routine.
+
+    Upon return from a \FUNC{shmem\_alltoall} routine, the following is true for
+    the local PE:
+    \begin{itemize}
+        \item Its \VAR{dest} symmetric data object is completely updated and the 
+	data has been copied out of the source data object.
```

Comment:
```suggestion
        \item Its \VAR{dest} symmetric data object is completely updated and the 
	data has been copied out of the \VAR{source} data object.
```

Reply:
@davidozog Were there any other instances where we need to make the same change?

Reply:
Also, should we use `\dest{}` and `\source{}` instead of `\VAR{...}`?